man/runc-list.8.md

```diff
@@ -4,12 +4,9 @@
 # SYNOPSIS
    runc list [command options] [arguments...]
 
-# DESCRIPTION
-   The default format is table.  The following will output the list of containers
-in json format:
-
-    # runc list -f json
-
 # OPTIONS
    --format, -f         select one of: table or json.
+                        The default format is table. The following will output
+                        the list of containers in json format:
+                        # runc list -f json
```

Comment:
This and `runc-ps.8.md` are inconsistent. I think that we should remove the "the following will ..." part because it's not necessary (the information is already there). Just make it clear that `table` is the default. And do the same for `runc ps`.